server/server.go

```diff
@@ -529,8 +531,10 @@ func (ss *subStore) Remove(cs *stores.ChannelStore, sub *subState, unsubscribe b
 			storageUpdate = sub.LastSent == qs.lastSent
 			sortedPendingMsgs := makeSortedPendingMsgs(sub.acksPending)
 			for _, pm := range sortedPendingMsgs {
-				m := cs.Msgs.Lookup(pm.seq)
-				if m == nil {
+				m, err := cs.Msgs.Lookup(pm.seq)
+				if m == nil || err != nil {
+					ss.stan.log.Errorf("Unable to update subscription for %s:%v (%v)",
+						sub.subject, pm.seq, err)
```

Comment:
We either need to panic/fatal, since we can't "move" this message to a different queue member, or log and continue. Same happens few lines below with the call to `AddSeqPending`.